cubedash/testutils/database.py

```diff
@@ -19,129 +20,252 @@
 from datacube.model import MetadataType
 from datacube.utils import read_documents
 from datacube.utils.documents import InvalidDocException, UnknownMetadataType
+from psycopg2 import sql
 from sqlalchemy import text
 
+logger: structlog.BoundLogger = structlog.get_logger(__name__)
+
+
 GET_DB_FROM_ENV = "get-the-db-from-the-environment-variable"
+POSTGIS_IMAGE = "postgis/postgis:16-3.5"
```

Comment:
I'm a bit afraid of having many places to update for the same piece of information because they tend to get out of sync, can this be read from `docker-compose.yaml` instead so things can't go out of sync?

Reply:
Agreed.

I'll find somewhere better for this, an environment variable, or a pytest argument, or reading from a config, or something.